code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1075,13 +1075,27 @@ Traitors and the like can also be revived with the previous role mostly intact.
 
 	SSblackbox.record_feedback("nested tally", "admin_toggle", 1, list("Toggled Hub Visibility", "[GLOB.hub_visibility ? "Enabled" : "Disabled"]")) //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/spiderize_more(mob/living/target as mob, progress = 1)
+	target.SetAllImmobility(500)
+	target.do_jitter_animation(progress * 200)
+	target.visible_message("<span class='danger'>Spiders emerge from [target.name]!</span>", "<span class='userdanger'>They crawl within you.</span>")
+	for (var/i in 1 to rand(progress * 9, progress * 11))
+		var/obj/structure/spider/spiderling/S = new(target.drop_location())
+		S.directive = "Erase the legacy of [target.name]."
+		QDEL_IN(S, rand(50, 300)) //just enough time for them to skitter around a little
+	if (progress < 5)
+		addtimer(CALLBACK(src, /client/proc/spiderize_more, target, progress + 1), 75)
+	else
+		target.visible_message("<span class='danger'>[target.name] explodes into a chittering mass!</span>", "<span class='userdanger'>The gods have rended your form.</span>")
+		target.gib(TRUE)
```

Comment:
1. This should really be a proc on the mob, not on the client of the admin
2. We only use the `as` modifier thing for procs which are called directly from the right hand panel(verbs)
3. Thats a bit too many spiders... 30 to 68 spiders is more reasonable than 135 to 165 spiders

```suggestion
/mob/living/carbon/proc/spiderize(progress = 1)
	SetAllImmobility(500)
	do_jitter_animation(progress * 200)
	visible_message("<span class='danger'>Spiders emerge from [target.name]!</span>", "<span class='userdanger'>They crawl within you.</span>")
	for (var/i in 1 to rand(progress * 3, progress * 5))
		var/obj/structure/spider/spiderling/S = new(drop_location())
		S.directive = "Erase the legacy of [target.name]."
		QDEL_IN(S, rand(50, 300)) //just enough time for them to skitter around a little
	if (progress < 5)
		addtimer(CALLBACK(src, /mob/living/carbon.proc/spiderize, progress + 1), 75)
		return
	visible_message("<span class='danger'>[target.name] explodes into a chittering mass!</span>", "<span class='userdanger'>The gods have rended your form.</span>")
	gib(TRUE)
```